e2e/helpers.go

```diff
@@ -16,18 +16,42 @@
 package e2e
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/google/uuid"
 
 	apitypes "github.com/openclarity/vmclarity/api/types"
 	"github.com/openclarity/vmclarity/core/to"
+	"github.com/openclarity/vmclarity/e2e/config"
+	"github.com/openclarity/vmclarity/testenv/types"
 )
 
-const (
-	DefaultScope   string = "assetInfo/labels/any(t: t/key eq 'scanconfig' and t/value eq 'test')"
-	DefaultTimeout        = 60 * time.Second
-)
+const DefaultTimeout = 60 * time.Second
+
+func getDefaultScope(cfg *config.Config) string {
+	scope := "assetInfo/%s/any(t: t/key eq 'scanconfig' and t/value eq 'test')"
+	switch cfg.TestEnvConfig.Platform {
+	case types.EnvironmentTypeAWS, types.EnvironmentTypeGCP, types.EnvironmentTypeAzure:
+		return fmt.Sprintf(scope, "tags")
+	case types.EnvironmentTypeDocker, types.EnvironmentTypeKubernetes:
+		return fmt.Sprintf(scope, "labels")
+	default:
+		return ""
+	}
+}
```

Comment:
```suggestion
func getDefaultScope(t types.EnvironmentType) string {
	scope := "assetInfo/%s/any(t: t/key eq 'scanconfig' and t/value eq 'test')"
	switch t {
	case types.EnvironmentTypeAWS, types.EnvironmentTypeGCP, types.EnvironmentTypeAzure:
		return fmt.Sprintf(scope, "tags")
	case types.EnvironmentTypeDocker, types.EnvironmentTypeKubernetes:
		return fmt.Sprintf(scope, "labels")
	default:
		return ""
	}
}
```
There is not need to pass the whole `Config` struct when only a single parameter is needed.